gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -605,111 +568,148 @@ function SWEP:Initialize()
 	end
 end
 
-function SWEP:CalcViewModel()
-	if not CLIENT or not IsFirstTimePredicted() then return end
+if SERVER then
+	local SF_WEAPON_START_CONSTRAINED = 1
 
-	self.bIron = self:GetIronsights()
-	self.fIronTime = self:GetIronsightsTime()
-	self.fCurrentTime = CurTime()
-	self.fCurrentSysTime = SysTime()
-end
+	---
+	-- Picked up by player. Transfer of stored ammo and such.
+	-- @param Player newowner
+	function SWEP:Equip(newowner)
```

Comment:
This should also be shared again, since it was before, and it might be used in a predicted / shared function. Otherwise this could break existing systems.